src/main/java/javax/jmdns/impl/JmDNSImpl.java

```diff
@@ -468,6 +468,7 @@ private void openMulticastSocket(HostInfo hostInfo) throws IOException {
             logger.trace("Trying to joinGroup({}, {})", multicastAddr, hostInfo.getInterface());
 
             // this joinGroup() might be less surprisingly so this is the default
+            _socket.setNetworkInterface(hostInfo.getInterface());
```

Comment:
Should this be better above the comment and also above the logging statement?

Reply:
Good point. Above the logging statement would be better.